### ISSUE TITLE
OG broken in Nuxt 3.8.1

### ISSUE THREAD

**Issue by bogdanciuca** [harlan-zw/nuxt-og-image#111]
### Describe the bug

Seems like the CSS isn't loaded anymore when the images get generated. 

## 3.8.0
![3.8.0](https://imgur.com/HIl3UJA.png)

## 3.8.1
![3.8.1](https://imgur.com/VbJ8Re6.png)

### Reproduction

Can add if needed

### System / Nuxt Info

```Shell
System:
    OS: macOS 13.6
    CPU: (8) arm64 Apple M1
    Memory: 53.42 MB / 8.00 GB
    Shell: 5.9 - /bin/zsh
Binaries:
    Node: 18.18.2 - ~/.nvm/versions/node/v18.18.2/bin/node
    npm: 10.2.3 - ~/.nvm/versions/node/v18.18.2/bin/npm
    pnpm: 8.10.5 - ~/.nvm/versions/node/v18.18.2/bin/pnpm
```

**Comment by bogdanciuca** [comment]
Update: Still reproducing with `nuxt: 3.8.2` and  `@nuxtseo/module: 2.0.0-beta.45`.

**Comment by harlan-zw** [comment]
Thanks will likely resolve this in v3 as I need to rework how the inline styling works.

If you need another work around in the mean time, you'll need to switch to inline styles / tailwind classes.

**Comment by bogdanciuca** [comment]
@harlan-zw Thanks for the response. Unfortunately I'm using vuetify instead of tailwind. Regarding inline styles, are you referring to `experimental.inlineSSRStyles: true` in nuxt config? I just tried it, with no difference.

**Comment by harlan-zw** [comment]
> @harlan-zw Thanks for the response. Unfortunately I'm using vuetify instead of tailwind. Regarding inline styles, are you referring to `experimental.inlineSSRStyles: true` in nuxt config? I just tried it, with no difference.

No, the issue is that Satori doesn't support non-inlined styles (or tailwind classes). 

To get around this limitation the module uses https://github.com/Stranger6667/css-inline, however, it seems like there's an issue with how it's extracting the styles with the latest Nuxt.

**Comment by bogdanciuca** [comment]
@harlan-zw Thanks for the context! It seems a bit weird that nuxt changed this behaviour in a patch version. I've created [a follow-up issue](https://github.com/Stranger6667/css-inline/issues/270) in the dependency repo.

**Comment by harlan-zw** [comment]
Are you able to share the template you're using? Need a good way to replicate this

**Comment by harlan-zw** [comment]
Should be performing better in `3.0.0-rc.21`, let me know if that's not the case.

**Comment by bogdanciuca** [comment]
@harlan-zw apologies for the delayed response, it's been a horrific end of the year. Will you be making a release of `Nuxt SEO` that includes this `OG` version?

**Comment by harlan-zw** [comment]
> @harlan-zw apologies for the delayed response, it's been a horrific end of year. Will you be making a release of `Nuxt SEO` that uses this `OG` version?

Sorry to hear that, don't feel any obligation to respond in a timely manner, it doesn't matter. Yes the latest beta of `@nuxtseo/module` uses it.

**Comment by bogdanciuca** [comment]
Thanks! 

Will you be publishing the RCs to the npm registry by any chance? Looks like latest published version is `2.0.0-beta.55`, which doesn't include the fix.
